src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -240,8 +242,9 @@
       <TestCommandLines Include="copy /y $(_TestILCFolder)\default.rd.xml  %EXECUTION_DIR%" />
       <TestCommandLines Include="rmdir /S /Q %EXECUTION_DIR%int" />
       <TestCommandLines Include="rmdir /S /Q %EXECUTION_DIR%native" />
+      <TestCommandLines Condition="'$(EnableMultifileTesting)' == 'true'" Include="call %RUNTIME_PATH%\TestILC\ilc.exe -buildSharedAssemblies -buildtype $(ILCBuildType) -v diag"/>
```

Comment:
>Are you sure you want to do this of every test 

It's not about whether we want it or not, when we pull the ILC toolchain we don't yet have the corefx runtime built so we can't do it at that time, and there  is no good central place to put it in order to run it once the runtime is there. It is definitelly something we can do eventually with some infrastructure changes, but this should be fine for now IMO.